src/domain/ledger/index.types.d.ts

```diff
@@ -34,6 +39,8 @@ type LedgerTransaction = {
 
   // for ln
   readonly paymentHash?: PaymentHash
+  readonly pubkey?: Pubkey
+  readonly feeKnownInAdvance?: boolean
```

Comment:
feeKnownInAdvance isn't undefined according to the [schema](https://github.com/GaloyMoney/galoy/blob/main/src/services/ledger/schema.ts#L71)

Reply:
There were instances in the db where a value wasn't set. Should we default that to `false` instead in the translation?